src/driver/driver.rs

```diff
@@ -1,14 +1,13 @@
 use url::Url;
 
-use error::{Error, NotFound};
+use error::Error;
 use query::{Condition, SortRule, Range, Query};
-use schema::Type;
-use value::{Value, ValueIter};
+use value::{Key, Value, Iter};
 
 /// The driver trait which all drivers will implement. Designed to be
 /// interoperable with any data source, however the driver also assumes a
 /// collection based data model.
-pub trait Driver {
+pub trait Driver: Send + Sync {
```

Comment:
It's required by `iron` which spawns multiple threads. The driver type (used inside all these threads) needs to have these traits.